.github/TEST_MATRIX.md

```diff
@@ -15,6 +15,8 @@ Molecule unit tests
 |![core/firewall unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/firewall%20unit%20testing/badge.svg)                       | x | x |   |
 |![core/log_client unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/log_client%20unit%20testing/badge.svg)                   | x | x | x |
 |![core/log_server unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/log_server%20unit%20testing/badge.svg)                   | x | x | x |
+|![core/nfs_client unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/nfs_client%20unit%20testing/badge.svg)                   | x | x | x |
```

Comment:
Can you uncheck OpenSUSE?

Reply:
Yes, I am going to correct that, I am not checking OpenSUSE.

Reply:
Corrected in commit f28b555